apps/web/components/PagesView/PagesView.vue

```diff
@@ -1,23 +1,22 @@
 <template>
-  <div class="relative">
-    <div class="pages-view sticky top-[52px] h-[calc(100vh-50px)] z-[2]" data-testid="pages-management-drawer">
-      <header class="flex items-center justify-between px-4 py-5 border-b">
-        <div class="flex items-center text-xl font-bold">
-          {{ getEditorTranslation('label') }}
-          <SfTooltip
-            :label="getEditorTranslation('open-manual-tooltip')"
-            placement="right"
-            :show-arrow="true"
-            class="flex"
-          >
-            <SfIconHelp class="ml-2 cursor-pointer" @click="openHelpPage" />
-          </SfTooltip>
-        </div>
-        <button data-testid="pages-view-close" class="!p-0" @click="closeDrawer">
-          <SfIconClose />
-        </button>
-      </header>
-
+  <div class="pages-view sticky top-[52px] z-[2]" data-testid="pages-management-drawer">
+    <header class="flex items-center justify-between px-4 py-5 border-b">
+      <div class="flex items-center text-xl font-bold">
+        {{ getEditorTranslation('label') }}
+        <SfTooltip
+          :label="getEditorTranslation('open-manual-tooltip')"
+          placement="right"
+          :show-arrow="true"
+          class="flex"
+        >
+          <SfIconHelp class="ml-2 cursor-pointer" @click="openHelpPage" />
+        </SfTooltip>
+      </div>
+      <button data-testid="pages-view-close" class="!p-0" @click="closeDrawer">
+        <SfIconClose />
+      </button>
+    </header>
+    <div class="h-[calc(100vh-150px)] overflow-y-auto">
```

Comment:
```suggestion
    <div class="h-[80vh] overflow-y-auto">
```

I reckon we should change it to `80vh` across all drawers? Otherwise we get stuff like this where the bottom may be cut off:

<img width="354" height="931" alt="Image" src="https://github.com/user-attachments/assets/2ff3dad6-d622-448b-a182-8174d125d833" />

cc. @abocsan-plenty

Reply:
looks good